shared/provision/forgot-username/container.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as ProvisionGen from '../../actions/provision-gen'
+import ForgotUsername from '.'
+import {connect} from '../../util/container'
+import {type RouteProps} from '../../route-tree/render-route'
+
+type OwnProps = RouteProps<{}, {}>
+
+const mapStateToProps = state => ({
+  forgotUsernameResult: state.provision.forgotUsernameResult,
+})
+
+const dispatchToProps = (dispatch, {navigateAppend, navigateUp}) => ({
```

Comment:
we should avoid making nav1 style stuff. prefer to use RouteTreeGen actions instead